### ISSUE TITLE
Why use a tuple in model.fit

### ISSUE THREAD

**Issue by MichaelangeloHuo** [woon9zhai/tf2.0_deepfm#1]
Why use a tuple in model.fit x=(train_feature_index.values,train_feature_value.values)
Does it make sense? thank you

**Comment by woon9zhai** [comment]
Because the input x of "model.fit()" must like follow: 
   x: Input data. It could be:
      - A Numpy array (or array-like), or a list of arrays
        (in case the model has multiple inputs).
      - A TensorFlow tensor, or a list of tensors
        (in case the model has multiple inputs).
      - A dict mapping input names to the corresponding array/tensors,
        if the model has named inputs.
      - A `tf.data` dataset. Should return a tuple
        of either `(inputs, targets)` or
        `(inputs, targets, sample_weights)`.
      - A generator or `keras.utils.Sequence` returning `(inputs, targets)`
        or `(inputs, targets, sample weights)`.

**Comment by MichaelangeloHuo** [comment]
Yes. I understand. Thank you. I am also wondering your inputs X; it consists of train_feature_index.values and train_feature_value.values.  Why would you separate two sets of inputs from the dataset, instead of one record one set of features (i.e., generally one input) ?
